batch/src/main/java/com/hartwig/batch/BatchArguments.java

```diff
@@ -41,8 +33,8 @@ static BatchArguments from(String[] args) {
             CommandLine commandLine = new DefaultParser().parse(options(), args);
             return ImmutableBatchArguments.builder()
                     .command(args[0])
-                    .project(commandLine.getOptionValue(PROJECT, "hmf-pipeline-development"))
-                    .region(commandLine.getOptionValue(REGION, "europe-west4"))
+                    .project(commandLine.getOptionValue(PROJECT, Arguments.DEFAULT_DEVELOPMENT_PROJECT))
```

Comment:
better to pull these up into CommonArguments than to introduce a dependency on pipeline Arguments